File: ToiLaHoi/Model/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ToiLaHoi.Model
{
    public class Product
    {
        public int  Id { get; set; }
        public Categories Categories { get; set; }
        public int CategoriesId { get; set; }
        [Required]
        public int Price { get; set; }
        public string ImageUrl { get; set; }
        [Required]
        [StringLength(255)]
        public string Title { get; set; }


    }
}

File: ToiLaHoi/ClientApp/app/model/product.ts



export interface Product {
    Id: number;
    Categories: Categories;

    Price: number;
    ImageUrl:string;
    Title: string;

}

export interface SaveProduct {
    Id: number;
    CategoriesId: string;

    Price: number;
    ImageUrl: string;
    Title: string;

}


export interface Categories {
    Name: string;
    Detail: string;
    Native: string;
    Id:number;
}



